generators/server/templates/_pom.xml

```diff
@@ -345,6 +345,13 @@
             <version>${gatling.version}</version>
             <scope>test</scope>
         </dependency>
+        <!-- include netty handler explicitly to overcome dependency mismatch when using cassandra -->
+        <%_ if (databaseType === 'cassandra') { _%>
+        <dependency>
+            <groupId>io.netty</groupId>
+            <artifactId>netty-handler</artifactId>
+            <version>4.0.36.Final</version>
+        </dependency><%_ } _%>
```

Comment:
could it be :
```
 </dependency>
<%_ } _%>
```

otherwise, I have this result:
```
        <dependency>
            <groupId>io.netty</groupId>
            <artifactId>netty-handler</artifactId>
            <version>4.0.36.Final</version>
        </dependency>        <dependency>
            <groupId>javax.inject</groupId>
            <artifactId>javax.inject</artifactId>
            <version>${javax.inject.version}</version>
        </dependency>
```